test/libsolidity/syntaxTests/parsing/conditional_with_assignment.sol

```diff
@@ -1,11 +1,10 @@
 contract A {
     function f() {
         uint y = 1;
-        uint x = 3 < 0 ? x = 3 : 6;
+        uint x = 3 < 0 ? y = 3 : 6;
         true ? x = 3 : 4;
     }
 }
 // ----
 // Warning: (17-119): No visibility specified. Defaulting to "public". 
```

Comment:
We could also get rid of the (unrelated) visibility and mutability warnings while we're at it. Same for some other tests. But fine in any case.

Reply:
I'd do that in a separate PR.